src/Test.UnitTests.Sarif.Multitool.Library/TestData/MergeCommand/Inputs/FileNameOnly.sarif

```diff
@@ -0,0 +1,360 @@
+          },
+          "locations": [{
+            "physicalLocation": {
+              "artifactLocation": {
+                "uri": "docker.io/fluxcd/flux:1.24.1",
+                "uriBaseId": "ROOTPATH"
+              }
+            }
+          }]
+        },
+        {
+          "ruleId": "docker.io/fluxcd/flux:1.24.1 (alpine 3.14.2): openssh-client-default-8.6_p1-r2 CVE-2021-41617",
+          "ruleIndex": 4,
+          "level": "error",
+          "message": {
+            "text": "sshd in OpenSSH 6.2 through 8.x before 8.8, when certain non-default configurations are used, allows privilege escalation because supplemental groups are not initialized as expected. Helper programs for AuthorizedKeysCommand and AuthorizedPrincipalsCommand may run with privileges associated with group memberships of the sshd process, if the configuration specifies running the command as a different user."
+          },
+          "locations": [{
+            "physicalLocation": {
+              "artifactLocation": {
+                "uri": "docker.io/fluxcd/flux:1.24.1",
+                "uriBaseId": "ROOTPATH"
+              }
+            }
+          }]
+        },
+        {
+          "ruleId": "docker.io/fluxcd/flux:1.24.1 (alpine 3.14.2): openssh-keygen-8.6_p1-r2 CVE-2021-41617",
+          "ruleIndex": 5,
+          "level": "error",
+          "message": {
+            "text": "sshd in OpenSSH 6.2 through 8.x before 8.8, when certain non-default configurations are used, allows privilege escalation because supplemental groups are not initialized as expected. Helper programs for AuthorizedKeysCommand and AuthorizedPrincipalsCommand may run with privileges associated with group memberships of the sshd process, if the configuration specifies running the command as a different user."
+          },
+          "locations": [{
+            "physicalLocation": {
+              "artifactLocation": {
+                "uri": "docker.io/fluxcd/flux:1.24.1",
+                "uriBaseId": "ROOTPATH"
+              }
+            }
+          }]
+        },
+        {
+          "ruleId": "usr/local/bin/kustomize: github.com/ulikunitz/xz-v0.5.5 CVE-2021-29482",
+          "ruleIndex": 0,
+          "level": "error",
+          "message": {
+            "text": "xz is a compression and decompression library focusing on the xz format completely written in Go. The function readUvarint used to read the xz container format may not terminate a loop provide malicous input. The problem has been fixed in release v0.5.8. As a workaround users can limit the size of the compressed file input to a reasonable size for their use case. The standard library had recently the same issue and got the CVE-2020-16845 allocated."
+          },
+          "locations": [{
+            "physicalLocation": {
+              "artifactLocation": {
+                "uri": "usr/local/bin/kustomize",
```

Comment:
>kustomize [](http://example.com/codeflow?start=38&length=9)

the name of this file is FileNameOnly, does that mean 'FileNameWithNoExtension'? or what? If it does mean this, can we rename this test file?

Reply:
no the case is the user pass FileName only and does not provide the path to the file.

the current name is very confusing, 
based on your input I think I should change it to like 'FileNameOnlyWithoutPath'

Reply:
updated the file name.